src/Microsoft.ML.Transforms/CustomMappingTransformer.cs

```diff
@@ -197,7 +197,7 @@ public sealed class CustomMappingEstimator<TSrc, TDst> : TrivialEstimator<Custom
         /// <param name="contractName">The contract name, used by ML.NET for loading the model. If <c>null</c> is specified, such a trained model would not be save-able.</param>
         /// <param name="inputSchemaDefinition">Additional parameters for schema mapping between <typeparamref name="TSrc"/> and input data.</param>
         /// <param name="outputSchemaDefinition">Additional parameters for schema mapping between <typeparamref name="TDst"/> and output data.</param>
-        public CustomMappingEstimator(IHostEnvironment env, Action<TSrc, TDst> mapAction, string contractName,
+        internal CustomMappingEstimator(IHostEnvironment env, Action<TSrc, TDst> mapAction, string contractName,
```

Comment:
>CustomMappingEstimator [](start = 17, length = 22)

We don't have any samples/ tests with it at all?

Reply:
I am adding a sample now. 

---
In reply to: [255816087](https://github.com/dotnet/machinelearning/pull/2510#discussion_r255816087) [](ancestors = 255816087)